test/browser-only/__snapshots__/constructLayoutEngine-browser.test.js.snap

```diff
@@ -33,9 +33,6 @@ exports[`constructLayoutEngine browser can successfully construct a layout engin
   <div
     id="single-spa-application:@org-name/footer"
   />
-  <div
```

Comment:
This snapshot demonstrates that the `app1` container element is removed now when the route is inactive